spec.html

```diff
@@ -703,7 +703,7 @@
           1. Set the running execution context's LexicalEnvironment to _lex_.
           1. <ins>Set the running execution context's PrivateNameEnvironment to _outerPrivateEnvironment_.</ins>
           1. Return Completion(_status_).
-        1. <ins>Append _fields_ to _fieldRecords_</ins>
+        1. <ins>Append to _fieldRecords_ the elements of _fields_.</ins>
```

Comment:
The English here seems awkward to me (I'd prefer "Append the elements of _fields_ to _fieldRecords_"), but it seems to match usage in other parts of ecma262, so I guess it's appropriate.